api_helper/api_helper.js

```diff
@@ -0,0 +1,20 @@
+import fetch from "node-fetch";
+
+export async function getPosts(url) {
+    const response = await fetch(url);
+    const results = await response.json();
+
+    return results
+}
+
+export async function postPosts(url) {
```

Comment:
```suggestion
export async function postPosts(url, requestBody) {
```
add body as a parameter